src/ant/README.md

```diff
@@ -22,16 +22,35 @@ Antenna violations can be repaired after global routing with the `repair_design`
 
 ## Commands
 
-### `check_antennas`
+```{note}
+- Parameters in square brackets `[-param param]` are optional.
+- Parameters without square brackets `-param2 param2` are required.
+```
 
-Check nets for antenna violations.
+### Check Antennas
 
+```tcl
+check_antennas 
+    [-net net] 
+    [-verbose]
 ```
-check_antennas [-net net] [-verbose]
-```
 
-- `-verbose` report all antenna calculations for violating nets
-- 'net' check antennas on one net
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-verbose` | Report all antenna calculations for violating nets. |
+| `-net` | Check antennas on specified net. |
```

Comment:
```suggestion
| `-net` | Check antennas on the specified net. |
```

Reply:
Can I clarify if you mean to remove all instances that has -report_file in the `check_antennas` function? Or just remove in the `define_cmd_args` section

Reply:
Remove only from the `sta:: define_cmd_args`.